### DISCUSSION TITLE
Fresh Google Chrome into the sandbox

### DISCUSSION THREAD

**Discussion by Bellisario** [sandboxie-plus/Sandboxie#1992] [Documents and notes]
How can I open Google Chrome in the sandbox without load current profiles, as a new fresh Chrome installation?
Currently when opening it on the sandbox I can see all my current profiles and so all my Chrome data...

![image](https://user-images.githubusercontent.com/72039923/175828180-c653ec86-f3b1-432c-b806-90eb0f32c7e0.png)

**Comment by DavidXanatos** [comment]
you can use a box set to privacy mode, or manually set the chrome profile paths as write only

**Reply by isaak654** [reply]
For further explanations, see the [PrivacyMode]( https://sandboxie-plus.github.io/sandboxie-docs/Content/PrivacyMode.html) documentation page.

**Comment by bjm234** [comment]
Hardened with Data Protection - UsePrivacyMode=y
as test: right click Edge desktop shortcut -> Run Sandboxed -> Hardened with Data Protection sbox [HardenedDP]
<img width="528" alt="png_15190" src="https://user-images.githubusercontent.com/25015553/175831579-817eb464-f716-4efd-acb1-7cfce9464bff.png">
1.1.3
W10 Home 21H2 (19044.1766)

**Comment by DavidXanatos** [comment]
the typical trouble shooting is to run edge without the privacy mode to create teh needed fodlers than close it and switch privacy mode on

**Reply by bjm234** [reply]
> the typical trouble shooting is to run edge without the privacy mode to create the needed folders then close it and switch privacy mode on

Okay.  Thanks

**Comment by bjm234** [comment]
Okay...after the needed folders are created with my Security Hardened Sandbox.
How may I retain the needed folders with auto-delete?

**Reply by hg421** [reply]
The snapshots feature could help, auto-delete should reset to the last snapshot

**Reply by bjm234** [reply]
> The snapshots feature could help, auto-delete should reset to the last snapshot

Snapshot does not retain my Gmail sign ins...head scratch.  Whereas Snapshot does retain GitHub sign in.   

==============================
Edit:  
I'm now remembering my posts in this thread were related to "with Data Protection" sbox.
